test/testhelper/testhelper.go

```diff
@@ -123,3 +123,20 @@ func Equals(tb testing.TB, exp, act interface{}) {
 		tb.FailNow()
 	}
 }
+
+// StringArrayEquivalent compares two string arrays and asserts that the arrays contain exactly the same items,
```

Comment:
unused?

Reply:
Lines 707 and 1932 in `pkg/question/question_test.go`.